web/src/site-admin/SiteAdminOverviewPage.tsx

```
@@ -93,13 +103,17 @@ export class SiteAdminOverviewPage extends React.Component<Props, State> {
         this.subscriptions.add(
             fetchWeeklyActiveUsers().subscribe(stats => this.setState({ stats }), error => this.setState({ error }))
         )
+        this.subscriptions.add(
+            fetchSiteAdminChecklist().subscribe(onboardingChecklist => this.setState({ onboardingChecklist }))
```

This swallows the error if the request fails